src/controllers/forecast.ts

```diff
@@ -41,8 +41,12 @@ export class ForecastController extends BaseController {
     res: Response
   ): Promise<void> {
     try {
+      const { orderBy } = req.query as { orderBy: 'asc' | 'desc' };
```

Comment:
Let's try to avoid `as`, we don't need to force types here, what about `const { orderBy } : { orderBy: 'asc' | 'desc' } = req.query` ?

Reply:
good, fixed.